project/src/main/ui/career/career-distance.gd

```diff
@@ -1,55 +0,0 @@
-extends HBoxContainer
-## UI elements for displaying and editing the player's distance in career mode.
-
-## Buttons for decreasing/increasing the player's distance
-onready var _down_button: Button = $Down
-onready var _up_button: Button = $Up
-
-func _ready() -> void:
-	PlayerData.career.connect("distance_travelled_changed", self, "_on_CareerData_distance_travelled_changed")
-	_refresh_buttons()
-
-
-## Refreshes the buttons based on the player's current distance.
-##
-## Buttons are disabled and hidden if the player can't decrease/increase their distance, either because they're at the
-## limits or because they're in the middle of a career session.
-func _refresh_buttons() -> void:
-	_down_button.disabled = false
-	_up_button.disabled = false
-	
-	if PlayerData.career.hours_passed > 0:
-		# disable the buttons; the player is in the middle of a career session
-		_down_button.disabled = true
-		_up_button.disabled = true
-	
-	var curr_region: CareerRegion = PlayerData.career.current_region()
-	var next_region: CareerRegion = PlayerData.career.next_region()
-	var prev_region: CareerRegion = PlayerData.career.prev_region()
```

Comment:
The 'next_region' and 'prev_region' methods are unused, and should be removed

Reply:
Removed `next_region()` and `prev_region()`. Verified there are no other references.